tests/test_deterministic_index/scripts/test.Rev

```diff
@@ -0,0 +1,34 @@
+seed(0)
+i ~ dnCategorical(v(0.5, 0.2, 0.3))
+for(k in 1:10)
+{
+  x[k] ~ dnNormal(0,1)
+}
+
+j := i*2
+y := x[i]
+z := x[j]
+w := x[2*i]
+
+model(i).graph("output/model.dot")
```

Comment:
Should `model.dot` be included in the expected output?

Reply:
I originally wanted it to be.  That is one way to ensure that the dependencies between `x[i]` and `i` are tracked.

But it turns out that names for the graph nodes are generated from pointers if the node is unnamed.  So the `model.dot` always changes even if the seed is constant.

In the future I would like to fix this, and then we can put `model.dot` in the expected output.  I will file an issue...